### DISCUSSION TITLE
PR created for older version

### DISCUSSION THREAD

**Discussion by alvindizon** [renovatebot/renovate#23572] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

This is the first time I've seen this behavior.

### Describe the problem

The latest version of [Picasso](https://github.com/square/picasso/releases), an image downloading library for Android applications, is `2.8`. However, our repo just received a PR for version `2.71828`. I don't have the time yet to create a repro of the said issue.

### Relevant debug logs
I can't seem to find the exact logs when the PR was created, as it was created 5 days ago and the most recent one I can see is 15 hours ago.
<details><summary>Logs</summary>

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
2.71828 sorts higher than 2.8

**Reply by JuJup** [reply]
I stumbled upon the same problem and I agree that it is a correct and plausible behaviour from Renovate. It seems that the developer of Picasso published a "false" version. So I would ask a different question. How can I ignore that specific version?
If I just close the MR which wants to update to 2.71828, there won't be new MR when there is Version 2.9 etc. if I understand correctly. 
So I guess this might the best solution? At least it seems to work.
```
"packageRules": [
    {
      "description": "Ignore invalid picasso version",
      "matchPackageNames": [ "com.squareup.picasso:picasso" ],
      "allowedVersions": "!/2.71828/"
    }
]
```
